### PR TITLE
update cirrus config to FreeBSD 13

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ task:
       - JULIA_VERSION: nightly
   allow_failures: $JULIA_VERSION == 'nightly'
   install_script:
-    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
+    - sh -c "$(https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
```
Comment (ararslan) [inline .cirrus.yml:12]:
```suggestion
    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
```

Comment (ararslan) [reply .cirrus.yml]: Looks like you caught that 🙂

Comment (dpo) [reply .cirrus.yml]: Thanks and sorry for the noise!

**Comment by dpo** [comment]
Closing as the FreeBSD version was not the source of the build failures.

File: .cirrus.yml (hunk 1 above)
Comment (ararslan) [reply .cirrus.yml]: No problem at all. I'm just happy that someone is actually using CirrusCI.jl 😄

**Pull request closed without merging**